### PR TITLE
[DOC] Add videos to docs

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:

Adds new videos to the Tempo docs for what are traces, traces vs los vs metrics, and intro to distributed tracing. 

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/tempo/configuration/_index.md
Comment (knylander-grafana) [inline docs/sources/tempo/configuration/_index.md:15]:
```suggestion
Instructions for configuring Tempo data sources are available in the [Grafana Cloud](/docs/grafana-cloud/send-data/traces/) and [Grafana](/docs/grafana/latest/datasources/tempo/) documentation.
```

File: docs/sources/tempo/getting-started/tempo-in-grafana.md
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/tempo-in-grafana.md:70]:
```suggestion
Trace to profiles lets you link your [Grafana Pyroscope](/docs/pyroscope/) data source to tracing data in Grafana or Grafana Cloud.
```

File: docs/sources/tempo/getting-started/tempo-in-grafana.md
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/tempo-in-grafana.md:73]:
```suggestion
For more information, refer to the [Traces to profiles documentation](docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#trace-to-profiles) and the [Grafana Pyroscope data source documentation](/docs/grafana/latest/datasources/grafana-pyroscope/).
```

File: docs/sources/tempo/getting-started/tempo-in-grafana.md
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/tempo-in-grafana.md:73]:
```suggestion
For more information, refer to the [Traces to profiles documentation](docs/grafana/latest/datasources/tempo/configure-tempo-data-source#trace-to-profiles) and the [Grafana Pyroscope data source documentation](/docs/grafana/latest/datasources/grafana-pyroscope/).
```

File: docs/sources/tempo/getting-started/tempo-in-grafana.md
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/tempo-in-grafana.md:73]: @jdbaldry  I am not sure why doc-validator is saying I have to use a relref shortcode for this URL. It started as a full URL, which is what writers' toolkit says to do. doc-validator said nope, change to /docs/..., so I did. Now it says, use a relref. 

What's the right thing to use here?

Comment (jdbaldry) [reply docs/sources/tempo/getting-started/tempo-in-grafana.md:73]: https://github.com/grafana/tempo/pull/3331

`doc-validator` 4.0.0 has the latest guidance. The current version in this repository's CI was released before the guidance changed.

https://grafana.com/docs/writers-toolkit/write/links/ is the correct guidance.

File: docs/sources/tempo/getting-started/tempo-in-grafana.md
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/tempo-in-grafana.md:73]: The doc-validator issue will be resolved once the other doc-validator Tempo PR is merged.

Comment (kvrhdn) [reply docs/sources/tempo/getting-started/tempo-in-grafana.md:73]: I've just merged https://github.com/grafana/tempo/pull/3331 🙂

**Review by kvrhdn — APPROVED** [review]
LGTM!

File: docs/sources/tempo/getting-started/tempo-in-grafana.md
Comment (knylander-grafana) [reply docs/sources/tempo/getting-started/tempo-in-grafana.md:73]: @jdbaldry I'm going to default to what we have in the Writers' Toolkit which says to use a full URL when linking to grafana.com pages. 
image

The doc-validator is reporting an issue that I need to use partial URLs.

Comment (jdbaldry) [reply docs/sources/tempo/getting-started/tempo-in-grafana.md:73]: The workflow running on this PR is out of date and is still running `doc-validator` v3.0.0 (https://github.com/grafana/tempo/actions/runs/7658560103/workflow?pr=3330).

Comment (jdbaldry) [reply docs/sources/tempo/getting-started/tempo-in-grafana.md:73]: Probably easiest to ignore it for now, merge this PR and know that future PRs should run with `doc-validator` v4.0.0.

**Comment by knylander-grafana** [comment]
Not going to backport this PR. When I tried the backport, there was content for the next release that was included. The video links in the Tempo docs will jsut wait for the next release.

**Pull request merged**
